src/App.xaml

```diff
@@ -14,6 +14,9 @@
                 <ResourceDictionary Source="/Styles/TextBlock.xaml" />
                 <ResourceDictionary Source="/Styles/Thickness.xaml" />
                 <ResourceDictionary Source="/Styles/WindowTitleBar_ThemeResources.xaml" />
+
+                <!-- Resources from other projects -->
+                <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/SetupFlow_ThemeResources.xaml" />

```

Comment:
What does the stupflow need from here?

Reply:
Good question and I should have mentioned this in my PR description as well.
I am starting to use the application scoped theme resources to register reusable constants and theme configurations. This encourages reusability, design consistency and customization.

Now, one way to do that is by creating different resource dictionary files and adding them directly to the `App.xaml` file. However, that can pollute the App.xaml if we have many! So instead I created the `SetupFlow_ThemeResources.xaml` which itself references the resource dictionaries from the setup flow project. Similar to how we organize Dependency Injection we have `services.AddSetupFlow()` which itself contains `services.AddCloneRepo()` etc...